integration_test/third_party_apps_data/applications/oracledb/centos_rhel/install

```diff
@@ -0,0 +1,49 @@
+set -e
+set -o pipefail
+
+## Install Oracle DB from package(s)
+INSTALL_DIR="$(mktemp --directory)"
+cd "${INSTALL_DIR}"
+
+PREINSTALL_RPM=oracle-database-preinstall-19c-1.0-1.el7.x86_64.rpm
+
+curl -o "${PREINSTALL_RPM}" https://yum.oracle.com/repo/OracleLinux/OL7/latest/x86_64/getPackage/"${PREINSTALL_RPM}"
+sudo yum -y localinstall "${PREINSTALL_RPM}"
+rm "${PREINSTALL_RPM}"
+
+ORACLE_DB_RPM=oracle-database-ee-19c-1.0-1.x86_64.rpm
+gsutil cp gs://stackdriver-test-143416-oracle/"${ORACLE_DB_RPM}" .
```

Comment:
Could you add a comment here:

# This RPM was downloaded from http://www.oracle.com/technetwork/indexes/downloads/index.html
# using martijnvans's Oracle account (any account should work).

Reply:
Better URL: https://www.oracle.com/database/technologies/oracle-database-software-downloads.html